src/opnsense/mvc/app/controllers/OPNsense/Kea/forms/dialogDdnsForwardDomain.xml

```
@@ -0,0 +1,27 @@
+<form>
+    <field>
+        <id>ddns_domains.name</id>
+        <label>Forward domain</label>
+        <type>text</type>
+        <help>Forward DDNS domain (e.g. example.com.)</help>
+    </field>
+    <field>
+        <id>ddns_domains.key_name</id>
+        <label>TSIG key</label>
+        <type>dropdown</type>
+        <help>Select an optional TSIG key to authenticate updates for this domain. Note: if a DNS server has a TSIG key configured, that server-specific key takes precedence over the domain key.</help>
+        <grid_view>
+            <label>TSIG key</label>
+        </grid_view>
+    </field>
+    <field>
+        <id>ddns_domains.dns_servers</id>
+        <label>DNS servers</label>
+        <type>select_multiple</type>
+        <help>Select one or more shared DNS servers to update for this domain.</help>
+        <grid_view>
+            <label>DNS servers</label>
+            <visible>true</visible>
```

visible true is the default, you only need to define visible false if you want to hide the column from the grid.